src/lib/Command/IbexaSetupCommand.php

```diff
@@ -30,80 +33,222 @@ class IbexaSetupCommand extends BaseCommand
+            $this->printNewLine($output);
+        }
 
-            $progressBar->start($productSpecificFiles->count());
+        $progressBar->finish();
+        $output->writeln("\nCopying product specific files");
+
+        $progressBar->start($productSpecificFiles->count());
+        $this->printNewLine($output);
+        foreach ($productSpecificFiles as $file) {
+            if (
+                !array_key_exists($file->getRelativePathname(), $commonFilePathNames)
+                && $fileSystem->exists($file->getRelativePathname())
+            ) {
+                $output->writeln(
+                    sprintf("File '%s' exists and has been overwritten", $file->getRelativePathname()),
+                    OutputInterface::VERBOSITY_VERBOSE
+                );
+            }
+
+            $fileSystem->copy($file->getPathname(), $file->getRelativePathname(), true);
+            $progressBar->advance();
             $this->printNewLine($output);
-            foreach ($productSpecificFiles as $file) {
-                if (
-                    !array_key_exists($file->getRelativePathname(), $commonFilePathNames)
-                    && $fileSystem->exists($file->getRelativePathname())
-                ) {
-                    $output->writeln(
-                        sprintf("File '%s' exists and has been overwritten", $file->getRelativePathname()),
-                        OutputInterface::VERBOSITY_VERBOSE
-                    );
-                }
-
-                $fileSystem->copy($file->getPathname(), $file->getRelativePathname(), true);
-                $progressBar->advance();
-                $this->printNewLine($output);
+        }
+
+        $progressBar->finish();
+
+        $output->writeln("\nPlatform.sh config files installed successfully");
+    }
+
+    private function installSolrConfigFiles(InputInterface $input, OutputInterface $output): void
+    {
+        $port = is_numeric($input->getOption('solr')) ? (int)$input->getOption('solr') : null;
+
+        if ($port === null) {
+            throw new InvalidArgumentException('Port must be proper number.');
+        }
+
+        $this->getIO()->write(
+            sprintf('Installing SOLR config files... (used PORT: %d)', $port),
+            true,
+            IOInterface::NORMAL
+        );
+        $fileSystem = new Filesystem();
+
+        $product = IbexaProductVersion::getInstalledProduct();
+        $version = IbexaProductVersion::getInstalledProductVersion();
+
+        $solrConfigFiles = $this->getSolrVersionSpecificFiles($product, $version);
+
+        if ($solrConfigFiles->count() === 0) {
+            throw new InvalidArgumentException(
+                sprintf('No SOLR configuration provided for %s (%s)', $product, $version
+            ));
+        }
+        if (!$fileSystem->exists('bin/tika-app-1.20.jar')) {
+            $this->getIO()->write('Downloading Tika App 1.20 ...', true, IOInterface::NORMAL);
+            file_put_contents(
```

Comment:
Since this is Composer plugin, you should rather use HttpDownloader instead of native PHP functions.
I think this could work:
```php
$this->getComposer()->getLoop()->getHttpDownloader()->copy(
    'https://archive.apache.org/dist/tika/tika-app-1.20.jar',
    'bin/tika-app-1.20.jar'
);
```